rubocop-airbnb/.rubocop.yml

```diff
@@ -1,6 +1,5 @@
 inherit_from:
   - .rubocop_airbnb.yml
-  - ./config/default.yml
```

Comment:
This did not contribute any configs that weren't already being pulled in by our injection code. I checked via  `bundle exec rubocop --config .rubocop.yml --show-cops` and the same number of cops were being used with and without it.